bundles/org.openhab.binding.publictransportswitzerland/README.md

```diff
@@ -0,0 +1,30 @@
+# Public Transport Switzerland Binding
+
+Connects to the "Swiss public transport API" to provide real-time public transport information. [Link to the API](https://transport.opendata.ch/)
+
+For example, here is a station board in HABPanel. (Download [here](https://github.com/StefanieJaeger/HABPanel-departure-board))
+
+![Departure board in HABPanel](doc/departure_board_habpanel.png)
+
+## Supported Things
+
+### Stationboard
+
+Upcoming departures for a single station. This is what you would usually see displayed at the train station.
```

Comment:
You could mention the ThingTypeUID. You'll need it when using textual configuration. There are many users who use textual configuration.